test/integration/commandline.py

```diff
@@ -88,25 +74,44 @@ def init(ctx: Test) -> Tinc:
         log.debug('got code %d, ("%s", "%s")', server.returncode, stdout, stderr)
         check.equals(code, server.returncode)
 
+    tinc_flags = (
+        (0, ("get", "name")),
+        (0, ("-n", "foo", "get", "name")),
+        (0, ("-nfoo", "get", "name")),
+        (0, ("--net=foo", "get", "name")),
+        (0, ("--net", "foo", "get", "name")),
+        (0, ("-c", "conf", "-c", "conf")),
+        (0, ("-n", "net", "-n", "net")),
+        (0, (f"--pidfile={pf}", f"--pidfile={pf}")),
+        (1, ("-n", "foo", "get", "somethingreallyunknown")),
+        (1, ("--net",)),
+        (1, ("--net", "get", "name")),
+        (1, ("foo",)),
+        (1, ("-c", "conf", "-n", "n/e\\t")),
+    )
+
     for code, flags in tinc_flags:
         node.cmd(*flags, code=code)
 
 
 def test_relative_path(ctx: Test, chroot: bool) -> None:
     """Test tincd with relative paths."""
 
-    foo = init(ctx)
+    # A hack to work around the 108-char limit on UNIX socket path length.
+    # Create a temporary directory, move configuration there, run tests, and move it back.
+    # We could use a temporary short path for the pidfile like other tests do, but the whole
+    # point of this test is checking that a relative path works, and it's likely to be too long.
```

Comment:
This comment is not correct, I guess it's from an earlier version of your patch?

I'm OK with the changes otherwise, but the comments made me think that you could also just create a symlink in /tmp to the test working directory, and change the `-c` option to point to the symlink; this would avoid having to physically move the PID file to a different location.

Reply:
Thanks. Good thing I forgot to remove the comment. That version didn't work on Windows because of the `.cmd` wrapper with its own set of paths.